docs/troubleshooting/general.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Identifying the problem"
+date: 2023-08-02
+summary: "Steps on how to diagnose problems during PCSX2 Setup"
+draft: false
+toc: true
+sidebar_position: 1
+---
+
+## General
+
+Hit a snag? Follow through this page to find out more more on how to gather more information about the issues! Especially useful for when you wanted to report them.
+
+### Collecting Emulog
+
+When reporting an issue, it is highly recommended that you also provide `emulog.txt` and GS Dump (For graphical glitches) so we can help you better!
+
+`emulog.txt` contains useful program logs for the team to investigate issues, please provide emulog whenever you're reporting issues.
+
+To get the emulog:
+
+- First enable logging by checking `Tools > Enable File Logging`.
+- Restart PCSX2.
+- Reproduce your issue.
+- Close PCSX2, _and do not open it again. **PCSX2 must be closed before you will be able to upload the emulog, and opening it again will remove any helpful information from the file**._
+
+<Image cols={7} src={require("./img/emulog.webp").default} />
```

Comment:
you're fine to pick any number (1-12) you want here, but 7 is a little strange. `6` would be half-page width for example.